pkg/flowaggregator/exporter/ipfix.go

```diff
@@ -144,7 +150,8 @@ func (e *IPFIXExporter) UpdateOptions(opt *options.Options) {
 		e.observationDomainID = genObservationDomainID(e.clusterUUID)
 	}
 	e.templateRefreshTimeout = opt.TemplateRefreshTimeout
-	klog.InfoS("New IPFIXExporter configuration", "collectorAddress", e.externalFlowCollectorAddr, "collectorProtocol", e.externalFlowCollectorProto, "sendJSON", e.sendJSONRecord, "domainID", e.observationDomainID, "templateRefreshTimeout", e.templateRefreshTimeout)
+	e.maxIPFIXMsgSize = int(opt.Config.FlowCollector.MaxIPFIXMsgSize)
```

Comment:
Any range for this? should we check if the value provided by the user is a reasonable value?

Reply:
I added validation code, using the same limits as in the go-ipfix library